packages/EasyDoctrine/src/DBAL/Types/CarbonImmutableDateTimeMicrosecondsType.php

```diff
@@ -5,14 +5,18 @@
 namespace EonX\EasyDoctrine\DBAL\Types;
 
 use Carbon\CarbonImmutable;
+use DateTime;
 use DateTimeImmutable;
 use DateTimeInterface;
+use DateTimeZone;
 use Doctrine\DBAL\Platforms\AbstractPlatform;
 use Doctrine\DBAL\Types\ConversionException;
 use Doctrine\DBAL\Types\DateTimeImmutableType;
 
 final class CarbonImmutableDateTimeMicrosecondsType extends DateTimeImmutableType
```

Comment:
How about applying a similar fix to `DateTimeImmutableMicrosecondsType`?

Reply:
Done.